### PR TITLE
feat: added related discussions feature

### PR DESCRIPTION
**Pull request by jordanh**
# Description

Adds related discussions

## Demo

Coming soon, embedding vectors are being calculated for demo ♻️ ...

## Testing scenarios

TK to write

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label https://github.com/ParabolInc/parabol/labels/Skip%20Maintainer%20Review if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by jordanh** [comment]
@mattkrick just rebased to master after merging `feat-embedder-service`

File: packages/server/graphql/mutations/helpers/handleCompletedStage.ts [hunk 1]
```diff
@@ -111,15 +112,16 @@ const handleCompletedRetrospectiveStage = async (
     }))
     await Promise.all([
       insertDiscussions(discussions),
-      addAIGeneratedContentToThreads(discussPhaseStages, meetingId, teamId, dataLoader)
+      addAIGeneratedContentToThreads(discussPhaseStages, meetingId, teamId, dataLoader),
+      generateRelatedDiscussions(meeting, teamId, dataLoader)
```
Comment (mattkrick) [inline packages/server/graphql/mutations/helpers/handleCompletedStage.ts:116]: Q: is the embedding created here the same as the embedding created when the meeting ends?

**Comment by mattkrick** [comment]
gonna take a crack at freshening this PR up, should be ready tomorrow

**Comment by jordanh** [comment]
Closing this PR as it has since been superceded (and shipped to production!)

**Pull request closed without merging**